analyses/molecular-subtyping-MB/02-compare-classes.Rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "Comparison of Expected and Observed MB Subtype Classification"
+output:
+  html_document:
+    df_print: paged
+params:
+  expected_input:
+    value: input/expected_class.rds
+  observed_input:
+    value: results/mb-classified.rds
+---
+  
+```{r include = FALSE}
+knitr::opts_chunk$set(comment = NA)
+getOption(x = 'DT.warn.size', default = FALSE)
+```
+
+```{r load_packages, echo = TRUE}
+# load libraries
+suppressPackageStartupMessages(library(tidyverse))
+suppressPackageStartupMessages(library(knitr))
+suppressPackageStartupMessages(library(DT))
+```
+
+```{r read_data, echo = TRUE}
+# read input
+exp.class <- readRDS('input/expected_class.rds')
+obs.class <- readRDS('results/mb-classified.rds')
+
+# format ambiguous expected classes 
+exp.class <- exp.class %>%
+  mutate(pathology_subtype = replace(pathology_subtype, 
+                                     pathology_subtype == "Group 3 or 4", "Group3, Group4")) %>%
+  mutate(pathology_subtype = gsub(" ", "", pathology_subtype))
+```
+
+```{r dt_function, echo = FALSE}
+# custom datatable function
+viewDataTable <- function(dat){
+  DT::datatable(dat,
+                rownames = FALSE,
+                filter = "bottom",
+                class = 'cell-border stripe',
+                options = list(pageLength = 5,
+                               searchHighlight = TRUE,
+                               scrollX = TRUE,
+                               dom = 'tpi',
+                               initComplete = JS("function(settings, json) {",
+                                                 "$(this.api().table().header()).css({'background-color':
+                                            '#004467', 'color': '#fff'});","}"))
+  )
+}
+```
+
+#### Details:
```

Comment:
Can you add information about how many samples have pathology subtype labels and (briefly) what the process is for obtaining those labels (for you, not what goes into that for pathology! 🙂 ) please?

Reply:
I would 'mirror' this information in the module README as you've done with these other points, too.

Reply:
Updated README with breakdown of samples and subtypes from path report:  
https://github.com/komalsrathi/OpenPBTA-analysis/tree/mb-class-nb/analyses/molecular-subtyping-MB#02-compare-classesrmd

Just unsure of what the second part means: `process of obtaining those labels`. I am using a file that @jharenza provided as input.

Reply:
I basically meant how do we get the labels from what I assume is a pathology report to the file you got and does it come from another database, for example. But you state it's from a pathology report, so I think that's sufficient for now. Thank you for the update!